src/components/ContributionsEpic.js.ts

```diff
@@ -0,0 +1,90 @@
+// The function returned here is going to be called on the platform and run
```

Comment:
What's up with the file extension of this file? Is it supposed to be `.js.ts`? 🤔

Reply:
It's a normal TS file, but it's the "js" part of the ContributionsEpic component. If we were using React to hydrate the component, this file wouldn't be needed.